tasks/build.yml

```diff
@@ -0,0 +1,63 @@
+---
+
+- name: Create Prometheus PHP FPM Exporter directories
+  become: yes
+  file:
+    path: "{{ item }}"
+    state: directory
+    owner: "{{ prometheus_phpfpm_exporter.user }}"
+    group: "{{ prometheus_phpfpm_exporter.group }}"
+    mode: 0750
+  with_items:
+    - "{{ prometheus_phpfpm_exporter.path.install }}"
+    - "{{ prometheus_phpfpm_exporter.path.log }}"
+
+- name: Copy wrapper shell script
+  become: yes
+  copy:
+    src: prometheus_exporter.sh
+    dest: "{{ prometheus_phpfpm_exporter.path.install }}/prometheus_exporter.sh"
+    owner: "{{ prometheus_phpfpm_exporter.user }}"
+    group: "{{ prometheus_phpfpm_exporter.group }}"
+    mode: 0755
+
+- name: Create Prometheus PHP FPM Exporter directory
+  become: yes
+  file:
+    path: "{{ prometheus_phpfpm_exporter.path.install }}/prometheus_phpfpm_exporter"
+    state: directory
+    owner: "{{ prometheus_phpfpm_exporter.user }}"
+    group: "{{ prometheus_phpfpm_exporter.group }}"
+
+- name: Download Prometheus PHP FPM Exporter binary
+  become: yes
+  get_url:
+    url: "{{ prometheus_phpfpm_exporter.download_url }}"
+    dest: "{{ prometheus_phpfpm_exporter.path.install }}/prometheus_phpfpm_exporter/phpfpm_exporter"
+    checksum: "{{ prometheus_phpfpm_exporter.checksum }}"
+    owner: "{{ prometheus_phpfpm_exporter.user }}"
+    group: "{{ prometheus_phpfpm_exporter.group }}"
+    mode: 0755
+
+- name: Set Prometheus PHP FPM Exporter defaults
+  become: yes
+  lineinfile:
```

Comment:
And since it's a template it will be need to be created during the configure stage.